src/ResourceManager/DataMigration/Commands.DataMigration/help/New-AzDataMigrationMongoDbCollectionSetting.md

```diff
@@ -0,0 +1,111 @@
+---
+external help file: Microsoft.Azure.Commands.DataMigration.dll-Help.xml
+Module Name: AzureRM.DataMigration
+online version: https://docs.microsoft.com/en-us/powershell/module/az.datamigration/New-AzDataMigrationMongoDbCollectionSetting
+schema: 2.0.0
+---
+
+
+# New-AzDataMigrationMongoDbCollectionSetting
+
+## SYNOPSIS
+Creates collection setting for migration according for the mongoDb migration
+
+## SYNTAX
+
+```
+New-AzDataMigrationMongoDbCollectionSetting -Name <Name> [-TargetRequestUnit <TargetRequestUnit>] [-CanDelete] [-ShardKey <ShardKey>]
+```
+
+## DESCRIPTION
+The New-AzDataMigrationMongoDbCollectionSetting cmdlet creates the migration setting object that specifies the throughput and delete behavior.
+The output the cmdlet is key value pair with name of the collection, and value of the setting. The output is used in assembling
+the database level settings for migration.
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> New-AzDataMigrationMongoDbCollectionSetting -Name myCollection -TargetRequestUnit 1000 -CanDelete -ShardKey "_id:-1,age:1,name"
```

Comment:
@haitsongmsft for the examples in the markdown files you're adding, would you mind providing the resulting output if the user were to run the command? This gives users a better idea of what properties can be found on the object returned from the command

Reply:
updated